app/scripts/TilesetFinder.js

```diff
@@ -188,7 +176,7 @@ class TilesetFinder extends React.Component {
   }
 
   handleSelect() {
-    const { selectedOptions } = ReactDOM.findDOMNode(this.multiSelect);
+    const { selectedOptions } = this.multiSelect;
```

Comment:
Just double checking: `ReactDOM.findDOMNode` is no longer needed?

Reply:
Nope.